lang/en.json

```
@@ -170,18 +170,76 @@
     "Vehicle": "Vehicle"
   },
   "Type": {
+    "Action": {
+      "Label": {
+        "one": "Action",
+        "other": "Actions"
+      }
```

I wonder whether we should do these slightly differently, since it is my understanding that this `PluralRules` style is only supposed to be used when the count is present in the localization (e.g. `2 actions`), and isn't necessarily accurate without the count.

```suggestion
      "Header": "Actions",
      "Label": "Action"
```